assets/js/googlesitekit/api/cache.js

```diff
@@ -22,12 +22,134 @@
 import { isEqual } from 'lodash';
 
 /**
- * Internal dependencies
+ * Prefix used for all Site Kit keys.
+ *
+ * Anything not using this key should not be touched by this library.
+ *
+ * @private
+ */
+export const STORAGE_KEY_PREFIX = 'googlesitekit_';
+
+const defaultOrder = [ 'localStorage', 'sessionStorage' ];
+let storageBackend;
+let storageOrder = [ ...defaultOrder	];
+/**
+ * Override the storage backend.
+ *
+ * Largely used for tests. Should not be used directly.
+ *
+ * @private
+ * @param {*} backend Backend to set for the cache.
+ * @return {void}
```

Comment:
If return is `void`, it should be omitted instead.

Reply:
Noted. I went through and changed this throughout the rest of the repo too.